src/components/Reusable/Button.js

```diff
@@ -0,0 +1,10 @@
+const Button = ({ onClick, id, classes, children }) => {
+
+    return (
+        <button id={id} onClick={onClick} className={classes}>
```

Comment:
good that you are passing custom `onClick` function here in component for reusability.